src/raftstore/store/peer.rs

```diff
@@ -1037,12 +1042,16 @@ impl Peer {
                 // have no effect.
                 self.proposals.clear();
             }
+            let now = monotonic_raw_now();
```

Comment:
It is expensive. Can we sample instead?

Reply:
maybe we can just observe one committed_entry for every ready_apply, is that acceptable?

Reply:
maybe we can just observe one committed_entry for every ready_apply, is that acceptable?

Reply:
TiKV invokes it frequently, so making a syscall is unacceptable. Instead, you can choose a less precise clock, like CLOCK_MONOTONIC_COARSE.